### ISSUE TITLE
Python3 

### ISSUE THREAD

**Issue by arvindsree** [pagekite/PyBreeder#3]
This works great in python2 but fails in python3 
exec __FILES SyntaxError: Missing parentheses in call to 'exec'. 
Would be great if it is compatible

**Comment by BjarniRunar** [comment]
Agree, thank you for filing this. This issue is now on my radar and I'll be working on it as time allows.

**Comment by pagekite** [comment]
This may have been fixed in a recent PR, but I haven't verified - because whether it is or isn't I think we are probably deprecating this tool (PyBreeder) and won't be fixing any more issues.

I recommend using [zipapp](https://docs.python.org/3/library/zipapp.html) instead, as it should be both more elegant and more performant.

Thank you for filing this and helping out!
